core/templates/css/oppia.css

```diff
@@ -4103,3 +4103,13 @@ oppia-response-text-input {
     display: none;
   }
 }
+
+.mat-icon.story-save-draft-warning {
+  color: #fc0;
```

Comment:
Please use the full code.

Reply:
The unnecessary modifications are the result of my vscode formatter. I have fixed them (yet to merge). The CSS linter that runs before bush give error for the full code, that's why I couldn't use it. Any idea why?

<img width="535" alt="image" src="https://user-images.githubusercontent.com/12639199/211730125-33851dbe-9212-40db-ac48-2e3501b73366.png">

Reply:
@vojtechjelinek I think this is actually enforced by our lint checks: see https://github.com/oppia/oppia/blame/develop/.stylelintrc#L22 and #12282

(I had also thought we use the full code, but apparently not...)

So I don't think a change is needed here...

Reply:
Ah yup, the same is stated in our style guide: https://github.com/oppia/oppia/wiki/Coding-style-guide#css